xml/System.IO.Compression/DeflateStream.xml

```diff
@@ -436,23 +447,32 @@ You use this constructor when you want to specify whether compression efficiency
             </Attribute>
           </Attributes>
         </Parameter>
-        <Parameter Name="state" Type="System.Object" Index="4" FrameworkAlternate="xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0" />
-      </Parameters>
       <Docs>
         <param name="buffer">The byte array to read the data into.</param>
         <param name="array">The byte array to read the data into.</param>
-        <param name="offset">The byte offset in <paramref name="array" /> at which to begin reading data from the stream.</param>
+        <param name="offset">The byte offset at which to begin reading data from the stream.</param>
         <param name="count">The maximum number of bytes to read.</param>
         <param name="asyncCallback">An optional asynchronous callback, to be called when the read operation is complete.</param>
-        <param name="cback">An optional asynchronous callback, to be called when the read operation is complete.</param>
```

Comment:
These are Xamarin-specific parameter names, and we no longer have Xamarin APIs in this repo.